Makefile

```diff
@@ -19,8 +19,12 @@ endif
 UNAME := $(shell uname)
 ARCH_UNAME := $(shell uname -m)
 LIBS = -l boost_program_options -l pthread -l z
-BOOST_INCLUDE = -I /usr/local/include/boost -I /usr/include
-BOOST_LIBRARY = -L /usr/local/lib -L /usr/lib
+ifeq ($(BOOST_INCLUDE),)
+  BOOST_INCLUDE = -I /usr/local/include/boost -I /usr/include
```

Comment:
getting this sorted for conda would be huge, although I'm wondering if it might be better to build vowpalwabbit as a conda package instead of trying to support installation from the source

some instructions here: https://conda.io/docs/user-guide/tutorials/build-pkgs.html

Reply:
Thanks, you are absolutely right, I did not think of other OS's. My bad. A simple work around would be to have different environmental variable names, e.g. VW_BOOST_XXX, to check them after OS-specific settings are done around [line 60](https://github.com/mlisovyi/vowpal_wabbit/blob/8fcea2734efa59a1c554e5bb13251569a65c61b2/Makefile#L60) and to overwrite the OS-specific values of BOOST_XXX that have been set.

Building a conda package would be ultimately the best solution. However, I have never done it before and I'm sure there will be some technical issues, which I do not have time to fight with right now. So if there is someone with the relevant  expertise and willing to dedicate a fraction of his/her time to it- wonderful.